config.h.in

```diff
@@ -9,6 +9,10 @@
 #define SAUNAFS_PACKAGE_VERSION_MICRO @PACKAGE_VERSION_MICRO@
 #define SAUNAFS_PACKAGE_VERSION "@PACKAGE_VERSION@"
 #define SAUNAFS_VERSHEX ((@PACKAGE_VERSION_MAJOR@)*0x10000+(@PACKAGE_VERSION_MINOR@)*0x100+(@PACKAGE_VERSION_MICRO@))
+#ifdef _WIN32
+#define WINDOWS_ClIENT_VERSION "@WINCLIENT_PACKAGE_VERSION@"
```

Comment:
Is the version of the Windows Client different than the other parts of the system?

Reply:
Yes, latest version is 1.0.4.

Reply:
We need to agree on how to handle this, because it can be confusing for the end user. Please, see the picture, the package version might be 1.*, but it installs client 4.*.
![image](https://github.com/leil-io/saunafs/assets/22012909/968bc464-3447-40d6-a57e-7c99f70e87d4)